File: py2many/list_untyped.cpp
#include <cppitertools/range.hpp> // NOLINT(build/include_order)
#include <vector>                 // NOLINT(build/include_order)
template <typename T0> List fill_in_list(T0 size) {
  std::vector<
      decltype(std::declval<typename decltype(range(size))::value_type>() + 1)>
      l = {};
  for (auto i : iter::range(size)) {
    l.push_back(i + 1);
  }
  return l;
}

File: py2many/list_untyped.go
package main

import (
iter "github.com/hgfischer/go-iter")



func FillInList[T0 any](size T0 any) List {
var l List = []None{}
for _, i := range iter.NewIntSeq(iter.Start(0), iter.Stop(size)).All() {
l = append(l, (i + 1));
}
return l}


